packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -352,9 +373,11 @@ export default class ImageEdit implements EditorPlugin {
         this.lastSrc = this.image.getAttribute('src');
 
         // Set image src to original src to help show editing UI, also it will be used when regenerate image dataURL after editing
+        this.image.style.visibility = 'hidden';
         this.image.src = this.editInfo.src;
```

Comment:
Do we still need to change original image's src?

Reply:
Yes, because when we applyFormat we compare the image edited with the previous src, so to make sure apply format works as expected we also need to change this src.